app/src/main/java/com/bignerdranch/android/criminalintent/CrimeAdapter.kt

```diff
@@ -3,10 +3,12 @@ package com.bignerdranch.android.criminalintent
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.ImageView
 import android.widget.TextView
 import android.widget.Toast
 import androidx.recyclerview.widget.RecyclerView
 import com.bignerdranch.android.criminalintent.models.Crime
+import android.text.format.DateFormat
```

Comment:
unused import

Reply:
also, wasn't the challenge formatting the dates? maybe you're doing another PR with that